tests/infrastructure/crds.go

```diff
@@ -22,27 +22,20 @@ package infrastructure
 import (
 	"context"
 
-	"kubevirt.io/kubevirt/tests/framework/kubevirt"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
 	v1ext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"kubevirt.io/kubevirt/tests/framework/matcher"
-
-	"kubevirt.io/client-go/kubecli"
-
 	crds "kubevirt.io/kubevirt/pkg/virt-operator/resource/generate/components"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
 )
 
 var _ = DescribeInfra("CRDs", func() {
-	var virtClient kubecli.KubevirtClient
-	BeforeEach(func() {
-		virtClient = kubevirt.Client()
-	})
-
-	It("[test_id:5177]Should have structural schema", func() {
+	It("[test_id:5177]Should have structural schema", decorators.Conformance, func() {
```

Comment:
sorry for not reviewing this earlier, but must this be a kubevirt end-to-end test? 
It is more of an integration tests with k8s. We don't have such a test category today, so I see why we should keep the test here. But I do not understand why it is a core functionality that should be tested by `Conformance`.

Reply:
My reasoning is that the integration with k8s is an (obvious) feature of KubeVirt.
It can be considered a first gate: KubeVirt CRDs should be properly installed and discovered.
Without them, the system does not work at all.
It also ensures stability, when tested with different providers.
Once the discussion ends, I will update the commit message (saving CI resources :slightly_smiling_face:).
Please let me know. Thanks

Reply:
> My reasoning is that the integration with k8s is an (obvious) feature of KubeVirt.

Correct, hence I am in favor of keeping this test. I'm not yet convinced why it is a core KubeVirt feature. Maybe you can elaborate on

> It also ensures stability, when tested with different providers.
? Is it reasonable for this test to fail on a new provider, while other (true e2e tests) pass?

> Once the discussion ends, I will update the commit message (saving CI resources 🙂).

The best way to save CI resources is to continue your effort of dropping pointless slow or fragile tests. 🙂

Reply:
> Is it reasonable for this test to fail on a new provider, while other (true e2e tests) pass?

If you only run conformance tests I think it could happen since not all the CRDs have a conformance test. 
Running all the tests suite I don't think so.
But this test is really cheap and it is targeted for the infrastructure.
I think it's valuable but if the majority says the contrary, I will drop it. :slightly_smiling_face:

Reply:
> I think it's valuable but if the majority says the contrary, I will drop it.

The test is valuable. I'm trying to convince myself if it is a necessary part of `Conformance`.

Reply:
Maybe this is redundant, I do remember that crds should be required to be structural now...

Reply:
@xpivarc what do you mean by that?